certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -160,9 +160,9 @@ def deploy_cert(self, domain, cert_path, key_path,
         stapling_directives = []
         if self.version >= (1, 3, 7):
             stapling_directives = [
-                ['\n', 'ssl_trusted_certificate', ' ', chain_path],
-                ['\n', 'ssl_stapling', ' ', 'on'],
-                ['\n', 'ssl_stapling_verify', ' ', 'on'], ['\n']]
+                ['\n    ', 'ssl_trusted_certificate', ' ', chain_path],
+                ['\n    ', 'ssl_stapling', ' ', 'on'],
+                ['\n    ', 'ssl_stapling_verify', ' ', 'on'], ['\n']]
```

Comment:
This is the arbitrary-but-common case. Sometimes the indentation will be wrong. Adding indentation matching to `_add_directive` is future work; I have a branch that starts to do that sort of stuff (and makes all of this whitespace in new directives obsolete) but it's going to be a lot of work, and not land in 0.9.0.